cinema/models.py

```diff
@@ -47,8 +55,16 @@ def __str__(self):
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
-    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+    movie = models.ForeignKey(
+        Movie,
+        on_delete=models.CASCADE,
+        related_name="movies"
```

Comment:
The `related_name` for the `movie` ForeignKey in `MovieSession` is set to 'movies', which conflicts with the `related_name` used in the `Movie` model's ManyToManyField. Consider using a unique `related_name` to avoid conflicts.